.github/pull_request_template.md

```diff
@@ -66,14 +66,14 @@ Operator (after pushing the merge commit)
 
 Operator (reindex) 
 
-- [ ] Purchased BQ slot committment in `dev`                <sub>or this PR does not require reindexing `dev`</sub>
-- [ ] Started reindex in `dev`                              <sub>or this PR does not require reindexing `dev`</sub>
-- [ ] Deleted BQ slot committment in `dev`                  <sub>or this PR does not require reindexing `dev`</sub>
-- [ ] Checked for failures in `dev`                         <sub>or this PR does not require reindexing `dev`</sub>
-- [ ] Purchased BQ slot committment in `prod`               <sub>or this PR does not require reindexing `prod`</sub>
-- [ ] Started reindex in `prod`                             <sub>or this PR does not require reindexing `prod`</sub>
-- [ ] Deleted BQ slot committment in `prod`                 <sub>or this PR does not require reindexing `prod`</sub>
-- [ ] Checked for failures in `prod`                        <sub>or this PR does not require reindexing `prod`</sub>
+- [ ] Purchased BQ slot committment in `dev`                <sub>or this PR does not require reindexing or does not target `dev`</sub>
```

Comment:
How is this an improvement? To me the old and the new wording are equivalent, but the new one is more verbose. Discuss in parking lot.

Reply:
@hannes-ucsc: "Ok to go with proposed change."